deployment/terraform/variables.tf

```diff
@@ -101,11 +102,13 @@ variable "rds_database_name" {
 }
 
 variable "rds_database_username" {
-  type = string
+  type      = string
+  sensitive = true
 }
 
 variable "rds_database_password" {
-  type = string
+  type      = string
+  sensitive = true
 }
```

Comment:
Changes that involve these variables appear like this in the Terraform output.

![image](https://user-images.githubusercontent.com/1774125/103677735-aa6a4000-4f50-11eb-85d3-1339c467ea23.png)